sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubsMetricsProviderTests.cs

```diff
@@ -0,0 +1,231 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Primitives;
+using Microsoft.Azure.WebJobs.Extensions.EventHubs.Listeners;
+using Microsoft.Azure.WebJobs.Host.TestCommon;
+using Microsoft.Azure.WebJobs.Logging;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using static Moq.It;
```

Comment:
Do we need this?   It looks like you're explicitly using `It` inline (and it would real oddly without doing so...)

Reply:
Looks like this is used for the original tests from EventHubScaleMonitorTests, specifically the `IsAny` method in line 50
` _consumerClientMock.Setup(client => client.GetPartitionPropertiesAsync(IsAny<string>()))`. Open to suggestions here if there's a better approach.

Reply:
I overlooked that instance.   Personally, I'd prefer to see us use that in the intended fluent form. `It.IsAny` so that it reads more easily for those not familiar with Moq.   That said, I don't feel strongly enough about it that I'd push back.

Reply:
Sounds good! Going to leave as is for now -- resolving conversation